scripts/bump-version/index.js

```diff
@@ -95,14 +101,25 @@ const main = (program) => {
     listAllVersions()
 }
 
-const updatePeerDeps = (pkgJson, newMonorepoVersion) => {
-    const peerDependencies = pkgJson.peerDependencies
-    if (!peerDependencies) return
+const updatePeerDeps = (pkgJsonOrPackageName, newVersion, isIndependent = false) => {
```

Comment:
Can we leave this in a state that was better than before by adding some block comments. Having that would help me understand what is happening in this function as well as for other reviewers

Reply:
Done